go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.6-beta
 	github.com/lightninglabs/lndclient v0.14.0-5
+	github.com/lightninglabs/loop/swapserverrpc v0.0.0-00010101000000-000000000000
```

Comment:
This could use a future tag (e.g. v1.0.0) we add when the PR is merged right?

Reply:
Yes, if `go mod` somehow allows it, I'd point this to a tag that doesn't exist yet so we can just create and push it once the PR is merged.

Reply:
TIL you can do that!